src/www/ui/search-helper.php

```diff
@@ -36,8 +36,8 @@
  *         contains uploadtree_pk, parent, upload_fk, pfile_fk, ufile_mode, and
  *         ufile_name
  */
-function GetResults($Item, $Filename, $Upload, $tag, $Page, $SizeMin, $SizeMax, $searchtype,
-                    $License, $Copyright, $uploadDao, $groupID, $PG_CONN, $limit = 100)
+function GetResults($Item, $Filename, $Upload, $tag, $Page = 1, $SizeMin, $SizeMax, $searchtype,
+                    $License, $Copyright, $uploadDao, $groupID, $PG_CONN, $limit)
```

Comment:
Please retain the default `100` for `$limit` as this will cause other places calling `GetResults` to fail.

Reply:
@GMishx but i am setting the default limit with the variable 
https://github.com/Shruti3004/FOSSology-REST-API/pull/13/files#diff-ffe87cc10eca84010d4e33dd0e3f9170b0bc9bc5fe686f43bb2536e0bda67342R96-R98

Reply:
The function is called at multiple locations like
- https://github.com/Shruti3004/FOSSology-REST-API/blob/332fde3c89b1dc90514fcacfe33e0020c9b65714/src/www/ui/search.php#L221
- https://github.com/Shruti3004/FOSSology-REST-API/blob/5ff2d524ed6c0f1c47a90a0f680bfcd1551abb3f/src/www/ui/api/Controllers/SearchController.php#L104
- https://github.com/Shruti3004/FOSSology-REST-API/blob/332fde3c89b1dc90514fcacfe33e0020c9b65714/src/www/ui/ajax-notice-files.php#L75

For them, `$limit` is a new parameter which they are not aware of. Either you will have to pass `100` in each location where the function is called or simply provide a default value to it and push it at the last and PHP will add 100 for you.

Reply:
@GMishx fixed it.